Bottom: 8d9d45a30c8914363c8ab2510d23e5477bcbe1e2
Top:    09bc532609a2f4c6d9423a71ee9e45ee20e52068
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 13:19:14 -0500

Refresh of add-buildbot-ptxdist-worker

---
diff --git a/roles/buildbot-worker-ptxdist/defaults/main.yml b/roles/buildbot-worker-ptxdist/defaults/main.yml
index c3e0956c..7fb5f918 100644
--- a/roles/buildbot-worker-ptxdist/defaults/main.yml
+++ b/roles/buildbot-worker-ptxdist/defaults/main.yml
@@ -23,7 +23,7 @@ config_hostdir: '{{ docker_restore_config_base_dir }}/{{ container_name }}'
 cache_path: '/cache'
 
 # name of the cache volume
-cache_volume: '{{ docker_name_prefix }}{{ hostname }}-cache'
+cache_volume: '{{ docker_name_prefix }}{{ container_hostname }}-cache'
 
 # environment passed to the container
 container_env:
@@ -33,8 +33,11 @@ container_env:
   BUILDBOT_WORKER_NAME: 'worker-ptxdist'
   BUILDBOT_WORKER_PASS: 'pass'
 
+# container_hostname
+container_hostname: '{{ role_name }}'
+
 # the name of the container being started
-container_name: '{{ docker_name_prefix }}{{ hostname }}'
+container_name: '{{ docker_name_prefix }}{{ container_hostname }}'
 
 # volumes mounted within the container
 container_volumes:
@@ -46,10 +49,7 @@ container_volumes:
 data_path: '/buildbot'
 
 # name of the data volume
-data_volume: '{{ docker_name_prefix }}{{ hostname }}-data'
-
-# hostname
-hostname: buildbot-worker-ptxdist
+data_volume: '{{ docker_name_prefix }}{{ container_hostname }}-data'
 
 # address of hsm cryptographic service
 hsm_host: '{{ansible_facts.default_ipv4.address}}'
@@ -62,10 +62,10 @@ image_args:
   PTK_HSM_HOST: '{{hsm_host}}'
 
 # directory of the image source
-image_dir: '{{ docker_projects_dir }}/{{ hostname }}'
+image_dir: '{{ docker_projects_dir }}/{{ container_hostname }}'
 
 # name of the image being built
-image_name: '{{ docker_registry_username }}/{{ hostname }}'
+image_name: '{{ docker_registry_username }}/{{ container_hostname }}'
 
 # repository URI of the image source
 image_repo: '{{ buildbot_worker_ptxdist_image_repo | default("https://github.com/novatechweb/docker-buildbot-worker-ptxdist.git") }}'
